examples/samplepkg/samplepkg/jsonschemas/base/record.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Record",
+    "type": "object",
+    "properties": {
+        "created": {
+            "type": "string"
+        },
+        "modified": {
+            "type": "string"
+        }
+    }
+}
```

Comment:
Do we want to introduce the notion of versioning already?  (`record-0.1.json`, `productset-0.9.json`...)